content/post/ddos-detection.md

```diff
@@ -0,0 +1,419 @@
++++
+title = "DDoS Attack Detection with Wallaroo: A Real-time Time Series Analysis Example"
+draft = false
+date = 2017-11-24T00:00:00Z
+tags = [
+    "wallaroo",
+    "windowing",
+    "partitioning",
+    "python",
+    "example",
+    "tutorial",
+    "ddos",
+    "timeseries",
+    "anomaly detection"
+]
+categories = [
+    "Windowing",
+    "Examples"
+]
+description = "Building a DDoS attack detector using timeseries analysis in real time with Wallaroo"
+author = "nisanharamati"
++++
+
+
+This post will go through a real-world use case for Wallaroo, our distributed data processing framework for building high-performance streaming data applications. We will construct a streaming DDoS attack detector, which consumes a stream of request logs from a large group of servers and uses statistical anomaly detection to alert us when a server is under attack.
+
+If you're new to Wallaroo and are interested in learning more about it before diving into today's example, you can check our [Go Python, Go! Stream Processing for Python][go python] post, where we introduce the Wallaroo Python API.
+
+The code for the example in this post can be found on [GitHub][ddos-detection], and can be run with the [current version of Wallaroo (0.2.2)](https://github.com/WallarooLabs/wallaroo/tree/0.2.2).
+
+## When Your Service Layer is Under Attack
+
+Running a large application over the internet is really great! You can reach a massive customer base, and provide them with your digital product in no time at all. Unless...
```

Comment:
these 2 paragraphs are very odd together.

Running a site on the internet is great, except when they don't work, cloudflare says that a DDOS is...

There's a missing thought here.

I think this would be better done more like:

"Along with the benefits you get in terms of reach when running an application over the Internet, you are also exposed to a variety of threats including a distributed denial of service attack." 

Then on to the cloudflare and drop the image entirely.